### DISCUSSION TITLE
Prepopulating the configuration with SSID and Password from stored file

### DISCUSSION THREAD

**Discussion by Kolkman** [khoih-prog/ESPAsync_WiFiManager#115] [Q&A]
Is there a way to --  when the config file has loaded successful and WIFI SSID and Password are read from the file -- use them as defaults in the portal?

**Comment by khoih-prog** [comment]
Check [**Populate portal wifi with saved credentials** #91](https://github.com/khoih-prog/ESP_WiFiManager/discussions/91)

> You really don't need to do anything for SSID/PWD fields. If leaving them blank, the stored SSIDs/PWDs will be auto reuse. The real reason for the design is that only displaying Credentials on CP when necessary to avoid Security risk (somebody sees your secrets !!!)
> 
> Try and see.

Anyway, I'll publish a new release with the optional feature, similar to  [**ESP_WiFiManager** v1.12.0](https://github.com/khoih-prog/ESP_WiFiManager/releases/tag/v1.12.0)

**Comment by khoih-prog** [comment]
Hi @Kolkman 

Please try the new [**ESPAsync_WiFiManager** v1.15.0](https://github.com/khoih-prog/ESPAsync_WiFiManager/releases/tag/v1.15.0)

---

### Releases v1.15.0

1. Optionally display Credentials (SSIDs, PWDs) in Config Portal. Check [**Populate portal wifi with saved credentials** #91](https://github.com/khoih-prog/ESP_WiFiManager/discussions/91) and [**Prepopulating the configuration with SSID and Password from stored file** #115](https://github.com/khoih-prog/ESPAsync_WiFiManager/discussions/115)
2. Display `Credentials` Hint on **Config Portal**
3. Periodic code **clean-up**
